flask_security/translations/fr_FR/LC_MESSAGES/flask_security.po

```diff
@@ -0,0 +1,329 @@
+#: flask_security/templates/security/login_user.html:3
+#: flask_security/templates/security/send_login.html:3
+msgid "Login"
+msgstr "Connexion"
+
+#: flask_security/forms.py:34 flask_security/templates/security/_menu.html:6
+#: flask_security/templates/security/register_user.html:3
+msgid "Register"
+msgstr "Inscription"
+
+#: flask_security/forms.py:35
+msgid "Resend Confirmation Instructions"
+msgstr "Renvoyer les instructions de confirmation"
+
+#: flask_security/forms.py:36
+msgid "Recover Password"
+msgstr "Récupérer le mot de passe"
+
+#: flask_security/forms.py:37
+msgid "Reset Password"
+msgstr "Réinitialiser le mot de passe"
+
+#: flask_security/forms.py:38
+msgid "Retype Password"
+msgstr "Confirmer le mot de passe"
+
+#: flask_security/forms.py:39
+msgid "New Password"
+msgstr "Nouveau mot de passe"
+
+#: flask_security/forms.py:40
+msgid "Change Password"
+msgstr "Changer le mot de passe"
+
+#: flask_security/forms.py:41
+msgid "Send Login Link"
+msgstr "Envoyer le lien de connexion"
+
+#: flask_security/templates/security/_menu.html:2
+msgid "Menu"
+msgstr "Menu"
+
+#: flask_security/templates/security/_menu.html:9
+msgid "Forgot password"
+msgstr "Mot de passe oublié"
+
+#: flask_security/templates/security/_menu.html:12
+msgid "Confirm account"
+msgstr "Confirmer le compte"
+
+#: flask_security/templates/security/change_password.html:3
+msgid "Change password"
+msgstr "Changer de mot de passe"
+
+#: flask_security/templates/security/forgot_password.html:3
+msgid "Send password reset instructions"
+msgstr "Envoyer les instructions de réinitialisation de mot de passe"
+
+#: flask_security/templates/security/reset_password.html:3
+msgid "Reset password"
+msgstr "Réinitialiser le mot de passe"
+
+#: flask_security/templates/security/send_confirmation.html:3
+msgid "Resend confirmation instructions"
+msgstr "Renvoyer les instructions de confirmation"
+
+#: flask_security/templates/security/email/change_notice.html:1
+msgid "Your password has been changed."
+msgstr "Votre mot de passe a été changé"
+
+#: flask_security/templates/security/email/change_notice.html:3
+msgid "If you did not change your password,"
+msgstr "Si vous n'avez pas changé votre mot de passe,"
+
+#: flask_security/templates/security/email/change_notice.html:3
+msgid "click here to reset it"
+msgstr "cliquez ici pour le réinitialiser"
+
+#: flask_security/templates/security/email/confirmation_instructions.html:1
+msgid "Please confirm your email through the link below:"
+msgstr "Merci de confirmer votre adresse email via le lien ci-dessous :"
+
+#: flask_security/templates/security/email/confirmation_instructions.html:3
+#: flask_security/templates/security/email/welcome.html:6
+msgid "Confirm my account"
+msgstr "Confirmer mon compte"
+
+#: flask_security/templates/security/email/login_instructions.html:1
+#: flask_security/templates/security/email/welcome.html:1
+#, python-format
+msgid "Welcome %(email)s!"
+msgstr "Bienvenue %(email)s !"
```

Comment:
This is intended, that is the [way to do in French](https://www.thoughtco.com/how-to-use-french-punctuation-4086509). Not sure if it's a non-breaking space though. @abulte can you check/fix if that's not the case?